### DISCUSSION TITLE
egs_phd

### DISCUSSION THREAD

**Discussion by leylex** [nrc-cnrc/EGSnrc#1115] [Questions] [answered]
I recently conducted a simulation of a mixed gamma source experiment involving a bottle containing radioactive mixture placed on a germanium detector within lead shielding. The simulation geometry was set up using ConeStack within an air box. I simulated the radioactive mixture by using source collection. My objective is to compare the simulation results with experimental data to assess counting efficiency. However, upon comparison, I noticed that the number of counts in the simulation output (dat file) is significantly lower than that observed in the experimental data. 

1. Was egs_phd user code the correct choice for my aim?
2. To accurately mimic the time frame of the laboratory experiment, would it be appropriate to use the activity or relative activity tag to achieve this? Is there an emission time parameter?
Thank you in advance!

**Comment by ftessier** [comment]
**1. Yes, egs_phd appears to be the optimal choice for replicating a Ge detector measurement.** This application tallies the energy deposited from all decay products within a single decay event. It operates under the assumption that the transport time for all particles is negligible compared to the detector's time resolution. That is, the detector does not discern individual particles originating from the same decay.

**2. Indeed, specifying the activity is necessary.** In Monte Carlo simulations, there exists no inherent "physical time" unit; instead, all parameters are normalized by the number of incident particles or, in the case of radionuclide sources, by the number of decays. Therefore, you need to specify the decay rate (activity), in order to compare with experimental data over a defined time span.

**Reply by leylex** [reply]
Thank you very much for you prompt answer! Could you please check is the activity specified correctly. I used the value for total activity. If that is not the reason behind the insufficient count number, maybe there is a mistake in the geometry. Here is part of the code: 

```ruby
:start source:
       name = mixture
       library = egs_source_collection
       source names = s1 s2 s3 s4 s5 s6 s7 s8 s9
       activity = 141201.8
       weights = 0.814 0.016 0.017 0.027 0.027 0.026 0.031 0.021 0.021
:stop source:
```

Thanks once again!

**Reply by ftessier** [reply]
I will let the expert @rtownson advise further!

**Reply by rtownson** [reply, marked as answer]
First, it's worth noting that egs_phd reports number of counts (not the number of counts per history) in the histogram, so it will change as you adjust `ncase`. When you use `egs_radionuclide_source`, `ncase` controls the number of particles emitted from the source, not the number of decays. To see the number of decays that were actually simulated, you can check `last case` in the egs_phd output.

`egs_source_collection` does not have an `activity` parameter. In [egs_radionuclide_source](https://nrc-cnrc.github.io/EGSnrc/doc/pirs898/classEGS__RadionuclideSource.html) you can set activity for each of your sources (not the total activity). However, all this does is sample/set the `emission time` parameter for source particles, which in turn is only used if you also set `experiment time`. The only functionality there is to throw away emissions that would occur outside your experiment duration. I expect this is not useful for you.

**The conclusion here is just to normalize the egs_phd number of counts to be per decay (divide by `last case`), and then scale it to the correct activity.**

Finally, you may want to check out the new [egs_gammaspec](https://github.com/nrc-cnrc/EGSnrc/pull/910) application. It does detector efficiency calculations, full energy peak efficiency, and coincidence corrections. Note that this is not in the EGSnrc release yet, it is sitting as a pull request. To test it out you'll have to use git to checkout the branch (`git checkout add-egs-gammaspec`), recompile egs++, copy the new egs_gammspec folder to egs_home and compile the application.

**Reply by leylex** [reply]
Dear @rtownson and @ftessier thank you for your insightful explanation; as someone new to EGS, your response is exactly what I needed.  I'll be sure to follow your recommendations.
